### PR TITLE
Random crop

### PR DESCRIPTION
**Pull request by szympie**

### CODE REVIEW

File: brats/transformations.py [hunk 1]
```diff
@@ -251,3 +234,55 @@ def _(mask: torch.Tensor, classes: typing.List[int]) -> torch.Tensor:
...
+    Interface for transformations that are supposed to transform multiple inputs in the same way.
+    """
+
+    @abc.abstractmethod
+    def __call__(self, imgs: typing.List[typing.Union[np.ndarray, torch.Tensor]]):
+        raise NotImplementedError
+
+
+class RandomCrop(CommonTransformation):
+    def __init__(self, size: typing.Tuple[int, int]):
```
Comment (myychal) [inline brats/transformations.py:250]: The user does not know whether passed tuple should be `(x, y)` or `(y, x)`. Docstring maybe?

File: brats/transformations.py [hunk 2]
```diff
@@ -251,3 +234,55 @@ def _(mask: torch.Tensor, classes: typing.List[int]) -> torch.Tensor:
...
+
+    def __init__(self, transforms: typing.List[CommonTransformation]):
+        self.transforms = transforms
+
+    def __call__(self, *imgs: typing.Union[np.ndarray, torch.Tensor]) -> typing.Tuple[
+        typing.Union[np.ndarray, torch.Tensor],
+        typing.Union[np.ndarray, torch.Tensor]]:
+        assert all(
+            img[0, 0, ...].shape == imgs[0][0, 0, ...].shape for img in imgs), "W and H of all images must be the same"
+        transformed = [utils.copy(img) for img in imgs]
```
Comment (myychal) [inline brats/transformations.py:285]: You first copy here, and then copy in call to RandomCrop. Is this much copying really necessary? Why is it necessary at all?

File: brats/transformations.py (hunk 1 above)
Comment (szympie) [reply brats/transformations.py:250]: Docstring added.

File: brats/transformations.py (hunk 2 above)
Comment (szympie) [reply brats/transformations.py:285]: I would like every entry to be individual entity, so the variables would not be changed by accident (by changing other variable referencing the same object).

**Pull request merged**